EIPS/eip-7701.md

```diff
@@ -107,7 +107,15 @@ The indices are as follows:
 | 0xf2 | `execution_gas_used`       | 32        |              | See transaction scoped vars in processing flow |
 | 0xff | `tx_hash_for_signature`    | 32        |              | Hash of the transaction without the signature  |
 
-## AA transaction processing flow
+
+### Costs of accessing cold addresses for Sender, Paymaster, and Deployer
+
+The Sender address is pre-warmed as part of the `AA_BASE_GAS_COST`.
+
+When a non-zero address that is not equal to the `Sender` address, is provided for a `Paymaster` or a `Deployer` contract,
+an additional EIP-2929 `COLD_ACCOUNT_READ_COST` cost of 2600 gas is charged and the address is added to `accessed_addresses`.
```

Comment:
paymaster, deployer are known beforehand, and thus should be treated as if they appeared in the access list. 
The EIP-2930 ACCESS_LIST_ADDRESS_COST is 2400